avm/res/storage/storage-account/main.bicep

```diff
@@ -529,7 +530,7 @@ type roleAssignmentType = {
 }[]?
 
 type networkAclsType = {
-  @description('Optional. Sets the resource access rules.')
+  @description('Optional. Sets the resource access rules. Array entries must consist of "tenantId" and "resourceId" fields only.')
```

Comment:
How about, instead of updating the description, you extend the UDT like
```bicep
type networkAclsType = {

  @description('Optional. Sets the resource access rules.')
  resourceAccessRules: {
    @description('Required. ...')
    tenantId: string

    @description('Required. ...')
    resourceId: string
  }[]?

  (..)
}
```

Reply:
Then you also get fancy linter support when using the parameter (like e.g., in the test file)

Reply:
And do I see it correct, the resourceId supports wild cards? 😄

Reply:
> And do I see it correct, the resourceId supports wild cards? 😄

yep, you can put a star more or less anywhere to e.g. allow allow container registries in a subscription to access the resource.

Reply:
added your changes.